docs/src/main/sphinx/release/release-414.md

```diff
@@ -20,10 +20,11 @@
 
 ## Security
 
-* Disallow writing system information with the `default` system access control.
-  System information writes can be re-enabled by setting the
-  `access-control.name` configuration property to `allow-all`, or by configuring
-  [system information rules](system-file-auth-system_information). ({issue}`17105`)
+* Disallow requesting [graceful shutdown](/admin/graceful-shutdown) with the
```

Comment:
Should this be "the ability to request" or something to that extent?
Also, "a graceful shutdown."

Reply:
Reworded .. but differently since "requesting a graceful shutdown" is wrong